### ISSUE TITLE
commons-io dependency is not shipped with icerberg-parquet

### ISSUE THREAD

**Issue by manishmalhotrawork** [apache/incubator-iceberg#174]
Presto uses iceberg-parquet module, and failed at runtime.

presto:default> insert into iceberg.default.sample4 values (1, 'foo');

```
Query 20190502_231211_00006_4ejma, FAILED, 1 node
Splits: 35 total, 1 done (2.86%)
0:01 [0 rows, 0B] [0 rows/s, 0B/s]
Query 20190502_231211_00006_4ejma failed: org/apache/commons/io/Charsets
```

and looks like, this is because of commons-io is not pulled by iceberg-parquet module.

Added this to the presto-iceberg to resolve it.

```
    <!-- https://mvnrepository.com/artifact/commons-io/commons-io -->
   <dependency>
        <groupId>commons-io</groupId>
        <artifactId>commons-io</artifactId>
        <version>2.4</version>
        <scope>runtime</scope>
    </dependency>
</dependencies>
```

Presto server side error is like this:
org.apache.iceberg.parquet.ParquetConversions uses it.

`java.lang.NoClassDefFoundError: org/apache/commons/io/Charsets
	at org.apache.iceberg.parquet.ParquetConversions.lambda$converterFromParquet$0(ParquetConversions.java:76)
	at org.apache.iceberg.parquet.ParquetConversions.fromParquetPrimitive(ParquetConversions.java:55)
	at org.apache.iceberg.parquet.ParquetMetrics.fromMetadata(ParquetMetrics.java:91)
	at org.apache.iceberg.parquet.ParquetWriteAdapter.metrics(ParquetWriteAdapter.java:50)
	at io.prestosql.iceberg.IcebergPageSink.finish(IcebergPageSink.java:163)
	at io.prestosql.spi.connector.classloader.ClassLoaderSafeConnectorPageSink.finish(ClassLoaderSafeConnectorPageSink.java:74)
	at io.prestosql.operator.TableWriterOperator.finish(TableWriterOperator.java:193)
	at io.prestosql.operator.Driver.processInternal(Driver.java:397)
	at io.prestosql.operator.Driver.lambda$processFor$8(Driver.java:283)
	at io.prestosql.operator.Driver.tryWithLock(Driver.java:675)
	at io.prestosql.operator.Driver.processFor(Driver.java:276)
	at io.prestosql.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:1075)
	at io.prestosql.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:163)
	at io.prestosql.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:484)
	at io.prestosql.$gen.Presto_308_25_gfb17b76____20190502_230954_1.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)`

**Comment by SurenNihalani** [comment]
This should be fixed in #230?

**Comment by rdblue** [comment]
Yes, this should be fixed.
